### DISCUSSION TITLE
Update the `updated_at` field of users, when any permissions/roles change

### DISCUSSION THREAD

**Discussion by Aniket-IN** [spatie/laravel-permission#1978] [General]
Hi,
I was wondering is there any way I can achieve something like
whenever any user's role/permissions get changed, the user's `updated_at` field also gets updated. 

Currently I'm doing 
```php
  $user->syncPermissions($request->permissions);
  $user->touch(); // to update the updated_at field
```
It works, but I've to call it every time I do some changes on permissions/roles. 
how can I automate this? 

Maybe something like this: https://laravel.com/docs/8.x/eloquent-relationships#touching-parent-timestamps

**Comment by erikn69** [comment]
Overwrite methods and add the touch
